<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite ignored="0" hostname="GC02RH2WYG8WPE" failures="0" tests="11" name="com.yourcompany.Tests.ContactFormTest" time="34.645" errors="11" timestamp="21 Apr 2020 19:23:23 GMT" skipped="0">
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="4.470">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;chrome&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest-1&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'chrome',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest-1'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.386">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;MicrosoftEdge&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest-1&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'MicrosoftEdge',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest-1'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.330">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;firefox&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest-1&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'firefox',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest-1'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.388">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;firefox&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;macOS 10.15&#039;,
 &#039;version&#039;: &#039;latest&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'firefox',
 'name': 'contactFormTest',
 'platform': 'macOS 10.15',
 'version': 'latest'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.365">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;MicrosoftEdge&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'MicrosoftEdge',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="tearDown" time="0.000">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.yourcompany.Tests.TestBase.tearDown(TestBase.java:133)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.410">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;chrome&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'chrome',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.326">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;chrome&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;macOS 10.15&#039;,
 &#039;version&#039;: &#039;latest-1&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'chrome',
 'name': 'contactFormTest',
 'platform': 'macOS 10.15',
 'version': 'latest-1'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.279">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;firefox&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;WIN10&#039;,
 &#039;version&#039;: &#039;latest&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'firefox',
 'name': 'contactFormTest',
 'platform': 'WIN10',
 'version': 'latest'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.328">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;chrome&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;macOS 10.15&#039;,
 &#039;version&#039;: &#039;latest&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'chrome',
 'name': 'contactFormTest',
 'platform': 'macOS 10.15',
 'version': 'latest'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
  <testcase classname="com.yourcompany.Tests.ContactFormTest" name="contactFormTest" time="3.363">
    <error message="Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username &#039;null&#039; and access key &#039;null&#039; to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{&#039;browserName&#039;: &#039;firefox&#039;,
 &#039;name&#039;: &#039;contactFormTest&#039;,
 &#039;platform&#039;: &#039;macOS 10.15&#039;,
 &#039;version&#039;: &#039;latest-1&#039;}
Build info: version: &#039;3.11.0&#039;, revision: &#039;e59cfb3&#039;, time: &#039;2018-03-11T20:26:55.152Z&#039;
System info: host: &#039;GC02RH2WYG8WPE&#039;, ip: &#039;192.168.1.107&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.13.6&#039;, java.version: &#039;13.0.2&#039;
Driver info: driver.version: RemoteWebDriver" type="org.openqa.selenium.WebDriverException">
      <![CDATA[org.openqa.selenium.WebDriverException: Unable to parse remote response: Misconfigured -- Sauce Labs Authentication Error.
You used username 'null' and access key 'null' to authenticate, which are not valid Sauce Labs credentials.

The following desired capabilities were received:
{'browserName': 'firefox',
 'name': 'contactFormTest',
 'platform': 'macOS 10.15',
 'version': 'latest-1'}
Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'
System info: host: 'GC02RH2WYG8WPE', ip: '192.168.1.107', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '13.0.2'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:111)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)
	at com.yourcompany.Tests.TestBase.createDriver(TestBase.java:116)
	at com.yourcompany.Tests.ContactFormTest.contactFormTest(ContactFormTest.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:75)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
]]>
    </error>
  </testcase> <!-- contactFormTest -->
</testsuite> <!-- com.yourcompany.Tests.ContactFormTest -->
